editor/src/clj/editor/workspace.clj

```diff
@@ -579,6 +587,38 @@ ordinary paths."
   (load-java-editor-plugins! workspace)
   (load-clojure-editor-plugins! workspace touched-resources))
 
+(defn- sync-snapshot-errors-notifications! [workspace old-errors new-errors]
+  (when (or old-errors new-errors)
```

Comment:
Errors are already either `nil` or non-empty though. Do you think it still makes sense to use extra guards in case we change how `:errors` are introduced into the snapshot?

Reply:
I don't feel super strongly about it either way. But I will recall this conversation when we trip a bug from this assumption in the future. 😅